modules/webrtc/webrtc_data_channel.h

```diff
@@ -33,6 +33,8 @@
 
 #include "core/io/packet_peer.h"
 
+#define WRTC_BUF "network/limits/webrtc/max_buffer_kb"
```

Comment:
I think it's better to rename `network/limits/webrtc/max_buffer_kb` to `network/limits/webrtc/max_channel_in_buffer_kb` so it's clearer what it does, since we cannot really control output buffer in HTML5 platform, and for consistency with WebSocket.
Also rename `WRTC_BUF` to `WRTC_IN_BUF`.